src/icalendar/prop/__init__.py

```diff
@@ -911,6 +911,389 @@ def from_jcal(cls, jcal_property: list) -> Self:
+        if len(fields) != 5:
+            raise ValueError(f"N must have exactly 5 fields, got {len(fields)}: {ical}")
+        return tuple(fields)
+
+    def __eq__(self, other):
+        """self == other"""
+        return isinstance(other, vN) and self.fields == other.fields
+
+    def __repr__(self):
+        """String representation."""
+        return f"{self.__class__.__name__}({self.fields}, params={self.params})"
+
+    from icalendar.param import VALUE
+
+    def to_jcal(self, name: str) -> list:
+        """The jCal representation of this property according to :rfc:`7265`."""
+        result = [name, self.params.to_jcal(), self.VALUE.lower()]
+        result.extend(self.fields)
+        return result
+
+    @classmethod
+    def from_jcal(cls, jcal_property: list) -> Self:
+        """Parse jCal from :rfc:`7265`.
+
+        Args:
+            jcal_property: The jCal property to parse.
+
+        Raises:
+            ~error.JCalParsingError: If the provided jCal is invalid.
+        """
+        JCalParsingError.validate_property(jcal_property, cls)
+        if len(jcal_property) != 8:  # name, params, value_type, 5 fields
+            raise JCalParsingError(
+                f"N must have 8 elements (name, params, value_type, 5 fields), "
+                f"got {len(jcal_property)}"
+            )
+        for i, field in enumerate(jcal_property[3:], start=3):
+            JCalParsingError.validate_value_type(field, str, cls, i)
+        return cls(
+            tuple(jcal_property[3:]),
+            Parameters.from_jcal_property(jcal_property),
+        )
+
+    @classmethod
+    def examples(cls) -> list[vN]:
+        """Examples of vN."""
+        return [cls(("Doe", "John", "M.", "Dr.", "Jr."))]
+
+
+class vOrg:
+    """vCard ORG (Organization) structured property per :rfc:`6350` `Section 6.6.4 <https://datatracker.ietf.org/doc/html/rfc6350.html#section-6.6.4>`_.
+
+    The ORG property specifies the organizational name and units associated with the vCard.
+
+    Its value is a structured type consisting of components separated by semicolons.
+    The components are the organization name, followed by zero or more levels of organizational unit names:
+    organization-name; organizational-unit-1; organizational-unit-2; ...
+
+    Semicolons are field separators and are NOT escaped.
+    Commas and backslashes within field values ARE escaped per :rfc:`6350`.
+
+    Examples:
+        A property value consisting of an organizational name,
+        organizational unit #1 name, and organizational unit #2 name.
+
+        .. code-block:: text
+
+            ORG:ABC\, Inc.;North American Division;Marketing
+
+        The same example in icalendar.
+
+        .. code-block:: pycon
+
+            >>> from icalendar.prop import vOrg
+            >>> org = vOrg(("ABC, Inc.", "North American Division", "Marketing"))
+            >>> org.to_ical()
+            b'ABC\\\\, Inc.;North American Division;Marketing'
```

Comment:
This is weird. Let's make this consistent between both the docstring and its rendering by using `r` on the docstring.
```suggestion
            b'ABC\\, Inc.;North American Division;Marketing'
```

Reply:
I don't think an admonition is needed, the example with `r` demonstrates it, and developers familiar with Python strings will understand.